specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2021-03-01-preview/mfe.json

```diff
@@ -0,0 +1,6068 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/JobBase"
+        },
+        "systemData": {
+          "description": "System data associated with resource provider",
+          "allOf": [
+            {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+            }
+          ],
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true,
+      "additionalProperties": false
+    },
+    "JobBaseResourceArmPaginatedResult": {
+      "description": "A paginated list of JobBase entities.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "An array of objects of type JobBase.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JobBaseResource"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "LabelingJobResource": {
+      "description": "Azure Resource Manager resource envelope.",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The resource URL of the entity (not URL encoded).",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the resource entity.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The resource provider and type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "Additional attributes of the entity.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/LabelingJob"
+            }
+          ],
+          "x-ms-discriminator-value": "Labeling",
+          "x-ms-client-flatten": true
+        },
+        "systemData": {
+          "description": "System data associated with resource provider",
+          "allOf": [
+            {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+            }
+          ],
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true,
+      "additionalProperties": false
+    },
+    "LabelingJobResourceArmPaginatedResult": {
+      "description": "A paginated list of LabelingJob entities.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "An array of objects of type LabelingJob.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LabelingJobResource"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "ExportSummary": {
```

Comment:
If this is returned via a GET it needs to look like a resource with top-level name, type, and id properties